xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/service/dto/DomainEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.icthh.xm.commons.domain.event.service.dto;
+
+import com.icthh.xm.commons.domain.event.domain.ValidFor;
+import lombok.Data;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.UUID;
+
+@Data
+public class DomainEvent {
+    private UUID id;
+    private String txId;
+    private Instant eventDate;
+    private String aggregateId;
+    private String aggregateType;
```

Comment:
also I suggest create contructor, with minimal required fields for creation

Reply:
currently created constructor with all required fields